gtfu/src/main/java/gtfu/TripReportData.java

```diff
@@ -21,12 +21,31 @@ public int compareTo(TripReportData o) {
         return start - o.start;
     }
 
-    public String toString() {
-        return headSign + " @ " + Time.getHMForMillis(start) + " - " + Time.getHMForMillis(start + duration);
+    // Combining route name with start time creates a trip name
+    public String getTripName() {
+        return getCleanRouteName() + " @ " + Time.getHMForMillis(start);
     }
 
     public boolean overlaps(TripReportData td) {
         return (td.start >= start && td.start < start + duration)
             || (start >= td.start && start < td.start + td.duration);
     }
+
+    // This logic performs cleanup on use cases that may or may not still be present. Consider removing.
```

Comment:
for comments that need followup ("consider removing") it's good to agree on searchable keywords. I like '### TODO', but the specifics don't matter as long as they're agreed-upon